Blueprints/BlueprintDefinitions/vs2022/AnnotationsFramework/template/src/BlueprintBaseName.1/CalculatorService/CalculatorService.cs

```diff
@@ -0,0 +1,33 @@
+namespace BlueprintBaseName._1
```

Comment:
nit: could we stick to using top-level statements?

Reply:
Done